src/plays/countries-statics/Country.jsx

```diff
@@ -1,12 +1,12 @@
-import React, { useEffect, useState } from "react";
+import React, { useCallback, useEffect, useState } from "react";
 
 export default function Country({ activeGeo }) {
   const [country, setCountry] = useState();
   const [isloading, setIsLoading] = useState(false);
   const [errorInfo, setErrorInfo] = useState(null);
-  const getCountry = () => {
+  const getCountry = useCallback((activeGeoVal) => {
```

Comment:
Not sure, why should I need to memorized  it?

Reply:
Same here..

Reply:
Does it have any warning if we don't use it? if yes, then what is the warning?

Reply:
@koustov , As I said on every render it will create a new function reference if do not warp it with useCallback.